docs/lockfile_generation.md

```diff
@@ -1,9 +1,17 @@
 # Lockfile Generation
 
-The Phylum CLI can generate a lockfile when it is given a manifest file.
+Lockfiles can be directly parsed and analyzed by Phylum's CLI. However since
+manifest files only specify loose version requirements, it is necessary for the
+CLI to internaly generate the corresponding lockfile.
```

Comment:
Expanded this based on what @kylewillmon said about a little introduction for this page. Still brief, but giving a bit more detail on when generation actually takes place.